app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -27,15 +31,41 @@
 
 public class GuessLocationActivity extends AppCompatActivity implements OnMapReadyCallback, View.OnClickListener, MapboxMap.OnMapClickListener{
 
+    private Intent intent;
+    private Bundle extras;
+
+    //Default parameters if there  isn't an extra attached to the intend of this activity (EPFL position)
+    final private double defaultGuessLat = 46.5197;
+    final private double defaultGuessLng = 6.5657;
+    final private double defaultCameraLat = 46.5197;
+    final private double defaultCameraLng = 6.5657;

```

Comment:
Since you picked the same values for the default case, why not store only two variables  ?